website/src/pages/webinars/preparing-for-the-next-log4j.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { FunctionComponent } from 'react'
+import { PageProps } from 'gatsby'
+
+import Layout from '../../components/Layout'
+import WebinarLayout from '../../components/webinars/WebinarLayout'
+
+const speakers = [
+    {
+        name: 'Jon Kohler',
+        title: 'Technical Director of Solution Engineering at Nutanix',
+        img: '/case-studies/jon-kohler.png',
+        bio: 'Jon is a leader in the Solutions Engineering practice at Nutanix. His team covers a variety of full-stack technical solutions, focusing on both Big Data and Healthcare solutions.',
+    },
+    {
+        name: 'Mike McLaughlin',
+        title: 'Principle Customer Engineer at Sourcegraph',
+        img: '/case-studies/mike-mclaughlin.png',
+        bio: 'Mike is a Principal Customer Engineer at Sourcegraph working with some of today’s largest brands to solve the challenges of big code, developer velocity, and fixing vulnerabilities.',
+    },
+]
+
+export const Webinar: FunctionComponent<PageProps> = props => (
+    <Layout
+        location={props.location}
+        meta={{
+            title: '[On-Demand Webinar] Learn how Nutanix remediated Log4j in 4 days',
+            description:
+                'On-Demand Webinar. Learn how Nutanix uses Sourcegraph to find and fix security vulnerabilities quickly across their code base.',
+            image: 'https://about.sourcegraph.com/sourcegraph-og.png',
```

Comment:
You can remove this since we have a default fallback that should be used 99% of the time unless stated as a requirement.

```suggestion
```